package.json

```diff
@@ -74,13 +69,13 @@
     "calculate-cache-key-for-tree": "^2.0.0",
     "cldr-core": "^43.1.0",
     "ember-auto-import": "^2.6.3",
-    "ember-cli-babel": "^7.26.11",
+    "ember-cli-babel": "^8.0.0",
```

Comment:
Main change seems to be dropping Node 14 support.

https://github.com/babel/ember-cli-babel/releases/tag/v8.0.0